node-onboarding/app.js

```diff
@@ -0,0 +1,60 @@
+import express from 'express';
+import fetch from 'node-fetch';
+import dotenv from 'dotenv';
+dotenv.config({ silent: true });
+const app = express();
+const port = 3000;
+
+const BASE_URL = 'https://api.github.com';
+
+const fetchData = async (url) => {
+  const response = await fetch(url, {
+    headers: {
+      Authorization: `token ${process.env.TOKEN}`,
+    },
+  });
+  if (response?.ok) {
+    return response.json();
+  }
+  throw response;
+};
+
+app.listen(port, () => {
+  console.log(`App listening on port ${port}!`);
+});
+
+app.get('/projects/:userName/:repository', async (req, res) => {
+  const { repository, userName } = req.params;
+  const { regex } = req.query;
+  const url = `${BASE_URL}/repos/${userName}/${repository}/projects`;
```

Comment:
I think this could be split in several functions first to request projects, then columns per project and then tasks per column.